content/en/docs/concepts/scheduling-eviction/eviction-policy.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Eviction Policy
+content_template: templates/concept
+weight: 60
+---
+
+<!-- overview -->
+
+This page is an overview of Kubernetes' policy for eviction.
+
+<!-- body -->
+
+## Eviction Policy
+
+The {{< glossary_tooltip text="Kubelet" term_id="kubelet" >}} can proactively monitor for and prevent total starvation of a
+compute resource. In those cases, the `kubelet` can reclaim the starved
+resource by proactively failing one or more Pods. When the `kubelet` fails
+a Pod, it terminates all of its containers and transitions its `PodPhase` to `Failed`.
+If the evicted Pod is managed by a Deployment, the Deployment will create another Pod
+to be scheduled by Kubernetes.
```

Comment:
Considering adding a what's next section to this page (especially since it's so short, and lots of the information still lives in the task page:)

```suggestion
to be scheduled by Kubernetes.

## {{% heading "whatsnext" %}}

- Read [Configure out of resource handling](/docs/tasks/administer-cluster/out-of-resource/) to learn more about eviction signals, thresholds, and handling.
```

Reply:
@celestehorgan Sure, I felt it to be short :)  i will commit this.

Reply:
done.